BuildAndTest.proj

```diff
@@ -43,9 +38,8 @@
   <Target Name="Test">
 
     <ItemGroup Condition="'$(CIBuild)' == ''">
-      <TestAssemblies 
-        Include="Binaries\$(Configuration)\**\*.UnitTests*.dll" 
-        Exclude="Binaries\Roslyn.Compilers.NativeClient.UnitTests.dll" />
+      <TestAssemblies Include="Binaries\$(Configuration)\**\*.UnitTests*.dll"
+                      Exclude="Binaries\$(Configuration)\**\Roslyn*Services*UnitTests*dll"/>
```

Comment:
Disabling that much testing seems dangerous.  Why can't this be fixed before hand?